command/agent/config.go

```diff
@@ -175,6 +175,9 @@ type ClientConfig struct {
 	// CpuCompute is used to override any detected or default total CPU compute.
 	CpuCompute int `mapstructure:"cpu_total_compute"`
 
+	// MemoryMB is used to override any detected or default total memory.
+	MemoryMB int `mapstructure:"total_memory"`
```

Comment:
Rename the config var `memory_total_mb` here for the property to make it really explicit what the unit is, and to match `cpu_total_compute`